.github/workflows/releaseWithCoreBundle.yml

```diff
@@ -1,5 +1,11 @@
 name: publish source-tracking-bundle
 on:
+  workflow_run:
+    workflows:
+      - publish
```

Comment:
I updated the workflow from create-github-release to publish, since publish is the action that publishes the npm package whereas create-github-release only creates a release tag on github AFAIK @CristiCanizales @mshanemc